eslintrc

```diff
@@ -44,6 +43,8 @@
         // We turned this off since it was too much work for too
         // little benefit, especially for one-line props.
         "react/jsx-sort-props": 0,
+        // We turned this off too as we didn't see an explicit benefit
+        "react/sort-prop-types": 0,
```

Comment:
It looks like this used to be set but you cleared it as part of this review -- is that right?